test/client/base_view.test.js

```diff
@@ -9,11 +9,18 @@ describe('Base/View', function () {
   var BaseView;
 
   before(function () {
+    global.document = {
+      createElement: sinon.stub().returns('<div></div>')
```

Comment:
yep, the view constructor now requires document.createElement.  I figured this was the best way to make that happen. :)